types/wheel/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "dt-header": false
```

Comment:
If I'm right, this rule shouldn't be disabled here for new packages, `v0.0` is good to go.

Reply:
Considering SemVer, I figured that it would be important to list the full version number since breaking changes can occur without incrementing the major or minor version number.
But if you insist that it's necessary to follow the tslinting rules exactly, then I'll make the changes so this can be accepted in to the repository asap.

Reply:
Considering SemVer, I think it is better to allow `v0.0.5` in https://github.com/Microsoft/dtslint